src/computation/computation.graphql

```diff
@@ -10,4 +10,41 @@ input CourseEntryInput {
 type CourseEntry {
     courseId: String!
     studyProgram: String!
-}
\ No newline at end of file
+}
+
+input CourseRecommendationRequest {
+    variant: String!
+    semesterKey: SemesterKeyInput!
+	selectedCourse: [CourseKeyInput!]! 
+}
+
+input SemesterKeyInput {
```

Comment:
this is the exact same as `CourseGroupInput` in course.graghql. Should we reuse that file? or do u wanna declare `SemesterKeyInput` here to make it readable?